packages/utils/upgrade/resources/codemods/5.0.0/sqlite3-to-better-sqlite3.json.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'node:path';
+
+import type { modules } from '../../../dist';
+
+/**
+ * This codemod runs on the package.json of the Strapi project and updates
+ * the sqlite dependency to better-sqlite3q
+ */
+const transform: modules.runner.json.JSONTransform = (file, params) => {
+  const { cwd, json } = params;
+
+  const rootPackageJsonPath = path.join(cwd, 'package.json');
+
+  if (file.path !== rootPackageJsonPath) {
+    return file.json;
+  }
+
+  const j = json(file.json);
+
+  const targetProperties = ['sqlite3', 'vscode/sqlite3', 'sqlite-legacy'];
+
+  targetProperties.forEach((targetProperty) => {
+    const oldSqliteDependency = `dependencies.${targetProperty}`;
+    if (j.has(oldSqliteDependency)) {
+      j.remove(oldSqliteDependency);
+      // TODO check this version when releasing V5
+      j.set('dependencies.better-sqlite3', '9.0.0');
```

Comment:
You don't have to set each time, only once after the forEach should be enough :slightly_smiling_face:

Reply:
Question though: What happens if the app already uses another version of better-sqlite3? Do we want to force ours or do we want to use the existing one? (in this case a simple check before setting would do)

Reply:
If it uses better-sqlite3 it would go inside the condition because it's not an `oldSqliteDependency`

Reply:
I changed it anyways, now it should handle that use cases